pkgs/top-level/all-packages.nix

```
@@ -3085,6 +3085,8 @@ with pkgs;
 
   artyFX = callPackage ../applications/audio/artyFX { };
 
+  astro-language-server = callPackage ../by-name/as/astro-language-server/package.nix { pnpm_8 = pnpm_9; };
```

```suggestion
  astro-language-server = callPackage ../by-name/as/astro-language-server/package.nix { pnpm = pnpm_9; };
```

also `pnpm` is `pnpm_9` so this whole thing can be dropped, and change `pnpm_8` to `pnpm` in package.nix